src/mavis/pairing/pairing.py

```diff
@@ -175,10 +178,14 @@ def equivalent(event1: BreakpointPair, event2: BreakpointPair, distances=None) -
         [
             abs(Interval.dist(event1.break1, event2.break1)) > max_distance,
             abs(Interval.dist(event1.break2, event2.break2)) > max_distance,
-            event1.data[COLUMNS.event_type] != event2.data[COLUMNS.event_type],
         ]
     ):
         return False
+    if (
+        matching_event_type == True
+        and event1.data[COLUMNS.event_type] != event2.data[COLUMNS.event_type]
```

Comment:
you don't need to access the data dict here you can just call the property directly

```
event1.event_type != event2.event_type
```

Reply:
Addressed in [960bd67](https://github.com/bcgsc/mavis/pull/336/commits/960bd67fdca774210a6cd5aac4c8c14bbe2a2a5f)